### ISSUE TITLE
Scrape STATISTICAL INFORMATION ON NEPALESE AGRICULTURE 2073/74 (2016/17)

### ISSUE THREAD

**Issue by ravinepal** [Code4Nepal/data#51]
@nikeshbalami - do you have this on Open Data Nepal? http://moad.gov.np/public/uploads/639899230-Statistical%20Information%20on%20Nepalese%20Agriculture%202073,74%20(2016,17).pdf

**Comment by nikeshbalami** [comment]
Yes, we do have most of this PDF datasets under Agriculture category: http://opendatanepal.com/dataset?oknp_category_label=Agriculture

But this are all 75 districts datasets, which can't be used for NepalMap because its 77 now.
